src/app/tests/suites/certification/Test_TC_DRLK_2_12.yaml

```diff
@@ -0,0 +1,435 @@
+                value: 0
+              - name: "Credentials"
+                value: []
+              - name: "CreatorFabricIndex"
+                value: 1
+              - name: "LastModifiedFabricIndex"
+                value: 1
+              - name: "NextUserIndex"
+                value: null
+
+    - label: "Create new PIN credential and lock/unlock user"
+      command: "SetCredential"
+      timedInteractionTimeoutMs: 1000
+      arguments:
+          values:
+              - name: "OperationType"
+                value: 0
+              - name: "Credential"
+                value: { CredentialType: 1, CredentialIndex: 1 }
+              - name: "CredentialData"
+                value: "123456"
+              - name: "UserIndex"
+                value: 1
+              - name: "UserStatus"
+                value: null
+              - name: "UserType"
+                value: null
+      response:
+          values:
+              - name: "Status"
+                value: 0
+              - name: "UserIndex"
+                value: null
+              - name: "NextCredentialIndex"
+                value: 2
+
+    - label: "Verify created PIN credential"
+      command: "GetCredentialStatus"
+      arguments:
+          values:
+              - name: "Credential"
+                value: { CredentialType: 1, CredentialIndex: 1 }
+      response:
+          values:
+              - name: "CredentialExists"
+                value: true
+              - name: "UserIndex"
+                value: 1
+              - name: "CreatorFabricIndex"
+                value: 1
+              - name: "LastModifiedFabricIndex"
+                value: 1
+              - name: "NextCredentialIndex"
+                value: null
+
+    - label:
+          "TH writes the RequirePINforRemoteOperation attribute value as False
+          on the DUT"
+      PICS: DRLK.S.F07 && DRLK.S.F00 && DRLK.S.A0033
+      command: "writeAttribute"
+      attribute: "RequirePINforRemoteOperation"
+      arguments:
+          value: false
+
+    - label:
+          "TH writes the RequirePINforRemoteOperation attribute value as False
+          on the DUT and Verify DUT responds with UNSUPPORTED_WRITE"
+      PICS: DRLK.S.F07 && DRLK.S.F00 && !DRLK.S.A0033
+      command: "writeAttribute"
+      attribute: "RequirePINforRemoteOperation"
+      arguments:
+          value: false
+      response:
+          error: UNSUPPORTED_WRITE
+
+    - label: "TH reads the RequirePINforRemoteOperation attribute from the DUT"
+      PICS: DRLK.S.F07 && DRLK.S.F00 && DRLK.S.A0033
+      command: "readAttribute"
+      attribute: "RequirePINforRemoteOperation"
+      response:
+          value: false
+
+    - label: "TH sends the Unbolt Door command to the DUT without PINCode"
+      PICS: DRLK.S.C27.Rsp
+      command: "UnboltDoor"
+      timedInteractionTimeoutMs: 1000
+
+    - label: "TH sends the Unbolt Door command to the DUT with valid PINCode"
+      PICS: DRLK.S.C27.Rsp
```

Comment:
Again, wrong PICS in test plan.

Reply:
https://github.com/CHIP-Specifications/chip-test-plans/pull/2801